content/tutorial-multisign.md

```diff
@@ -20,9 +20,11 @@ To use multi-signing:
 Availability of Multi-Signing
 -----------------------------
 
-Multi-signing is built into `rippled` starting with [version 0.31.0](https://wiki.ripple.com/Rippled-0.31.0). Because multi-signing is a change in transaction processing, the change is due to be enabled by an **Amendment** to the Ripple Consensus Protocol. This Amendment must be approved by a consensus of validators showing consistent support for the feature over a period of time. For more information, see [Amendments](concept-amendments.html).
+Multi-signing has been enabled by an [**Amendment**](concept-amendments.html) to the Ripple Consensus Protocol since 2016-06-27.
 
-If you want to test multi-signing before it becomes available in the production network, or without risking real money, you can do so by running `rippled` in [stand-alone mode](concept-stand-alone-mode.html) with the MultiSign feature enabled. To enable multi-signing for testing, add the following stanza to your `rippled.cfg`:
+If you want to use multi-signing with `rippled` in [stand-alone mode](concept-stand-alone-mode.html) or on a [parallel network](tutorial-rippled-setup.html#parallel-networks), you must either advance to ledger 256 (past the first amendment vote), or force the MultiSign feature to be enabled. You can check the status of the MultiSign amendment using the [`feature` command](reference-rippled.html#feature).
```

Comment:
according to cpp team, the 256-ledger thing doesn't work because you still have to wait 2 weeks for the amendments to be approved even in stand-alone mode. So forcing the command on with the `[features]` stanza is the only way to test them (if using a fresh ledger).

Technically, it's not stand-alone mode that's the problem, specifically, but the fresh ledger. If you run in stand-alone mode by loading a ledger downloaded from the production network, you won't have this problem.